website/docs/how-to/how-to-create-feature-toggles.md

```diff
@@ -131,12 +131,6 @@ This can be done at any point after you've created your flag.
 
 :::
 
-:::tip API: add variants
-
-Use the [create variants endpoint](/reference/api/unleash/overwrite-feature-variants) to create or overwrite existing variants in all environments. The payload should be your desired variant configuration.
-
-:::
```

Comment:
@melindafekete the problem with this is that the endpoint was removed, so there's no page for overwrite-feature-variants anymore. I believe this will require some rewriting of this specific page

Reply:
Yes, I just pushed a change with the link to strategy-variants and removing the image which was old